main/src/ca/uwaterloo/flix/language/phase/EntryPoints.scala

```diff
@@ -545,8 +584,11 @@ object EntryPoints {
             duplicatedHandlerErrors += EntryPointError.DuplicateDefaultHandler(handledSym, loc2, loc1)
         }
       }
-
-      (root, errors.toList.flatMap(_.toList) ++ duplicatedHandlerErrors)
+      val defsWithDefaultHandlers = ParOps.parMapValues(root.defs)(
+        defn =>
+          if (isEntryPoint(defn)(root)) wrapDefWithDefaultHandlers(defn, validHandlers, root) else defn
```

Comment:
It looks like this transforms all entry points, but suppose I call a function somewhere in my code which happens to be an entry point. It should really only be transformed where it's _used_ as an entry point, right?